src/views/product-integration/index.tsx

```diff
@@ -11,14 +11,15 @@ import { Integration } from 'lib/integrations-api-client/integration'
 import { Release, ReleaseComponent } from 'lib/integrations-api-client/release'
 import { useRouter } from 'next/router'
 import { useMemo } from 'react'
-import ReactMarkdown from 'react-markdown'
```

Comment:
@thiskevinwang I think we can remove this from dependencies too.

This should be the only place we were using this & I added `react-markdown` in our integrations PR myself (wasn't already in DevDot).

Reply:
`react-markdown` is still used by `VariableGroupList` currently. My thought was to leave it for now, at least until Variable Groups' README content are processed by our `next-mdx-remote` pipeline as well.

It seemed like high-effort, low-reward, so I didn't do that part yet last week

Reply:
Oh!  Yes okay -- let's do that my bad!